app/views/roles/edit.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2>Role:</h2>
+<%= bootstrap_form_for @role, :url=>role_management.role_path(@role) do |f| %>
```

Comment:
Would you mind updating the hash syntax in this file? The `:url =>` etc. kind of hurts to look at these days.

Reply:
nuh uh (ok)